jenkins/docker/docker-re-release.jenkinsfile

```diff
@@ -0,0 +1,59 @@
+lib = library(identifier: 'jenkins@5.7.1', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    options {
+        timeout(time: 2, unit: 'HOURS')
+    }
+    agent none
+    parameters {
+        choice(
+            name: 'PRODUCT',
+            choices: ['opensearch', 'opensearch-dashboards'],
+            description: "Choose the product type among OpenSearch / OpenSearch-Dashboards."
+        )
+        choice(
+            name: 'TAG',
+            choices: ['1', 'latest'],
```

Comment:
This should be 1 and 2, not latest.

Reply:
@peterzhuamazon Can't this value be 2.9.0, 1.3.11?

Reply:
I think this workflow is intended to concentrate only on the latest versions. So tag should do the work and update the major tag and latest images. Or providing the latest version number in each branch be it 1.x or 2.x should do the same some job as per library implementation.

Reply:
You dont need to be 2.9.0/1.3.11 as we only patch the latest supported version.
And major 1/2 tag already tag to the latest version.